api/src/main/java/org/pmiops/workbench/google/DirectoryServiceImpl.java

```diff
@@ -38,6 +38,8 @@
   private static final String ALLOWED =
       "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*()";
   private static final String APPLICATION_NAME = "All of Us Researcher Workbench";
```

Comment:
nit/aside: Is there an easy way to externalize these kinds of constants (like a `.properties file`), in case we or other OSS users want to use some of this code for another application, or somebody who's not a programmer needs to make changes?

Reply:
I don't know of an easy way, but I also don't think I agree there's merit to doing that if we easily could. It increases complexity and we don't currently have the OSS use case you're describing.